src/ui/components/shared/WorkspaceSettingsModal/OrganizationSettings.tsx

```diff
@@ -107,7 +107,11 @@ const OrganizationSettings = ({ workspaceId }: { workspaceId: string }) => {
   const disabled = !workspace.isOrganization;
 
   return (
-    <div className={classNames("space-y-4", { "text-gray-500": disabled })}>
+    <div
+      className={classNames("-mr-4 max-h-full space-y-4 overflow-y-auto pr-4", {
```

Comment:
Moved the scrolling support out of the general settings pane and into the pane that needs it. Other settings panes in personal settings already handle this correctly.